test/functional/runner/index.js

```diff
@@ -73,11 +73,6 @@ function parseTopologies(topologies) {
     return ['replicaset', 'mongos', 'single'];
   }
 
-  const idx = topologies.indexOf('single');
```

Comment:
because `runOn` allows to run against single topologies as well. My guess is that it was written this way because the transactions YAML tests were never meant to be run against standalones, but everything else now uses the same code path.